clients/geth/specular/sbin/start_indexer.sh

```diff
@@ -17,6 +18,7 @@ $GETH_SPECULAR_DIR/build/bin/geth \
     --rollup.coinbase=f39fd6e51aad88f6f4ce6ab8827279cfffb92266 \
     --rollup.l1endpoint 'ws://localhost:8545' \
     --rollup.l1chainid 31337 \
+    --rollup.sequencer-addr '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266' \
     --rollup.sequencer-inbox-addr '0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512' \
     --rollup.rollup-addr '0x5FC8d32690cc91D4c39d9d3abcBD16989F875707' \
```

Comment:
maybe orthogonal to your PR but do these need to be hardcoded here—we're dumping them to a file now right? can we read from that

Reply:
yes we can make an env file for that and actually we should do that for both this and docker compose file, but maybe in next pr